### DISCUSSION TITLE
Device Details

### DISCUSSION THREAD

**Discussion by krulci** [gucci-on-fleek/lockdown-browser#39] [Q&A] [answered]
Is it possible to make Windows Sandbox appears as a real installation?
The concern with this is that if you were to take a look at Device Manager
![image](https://github.com/gucci-on-fleek/lockdown-browser/assets/52474054/baf56ea1-8fcd-4f02-8059-98abf44cd768)

You would see a lot of properties very unique to Windows Sandbox.
I am not sure if RLB collects these information or not, but will be good if this can be genuine from start.

**Comment by gucci-on-fleek** [comment, marked as answer]
It's presumably possible to hide all of those devices, but it's likely quite difficult. Without doing a ton of research and testing, I have no idea how to hide those.


But if the _Browser_ knew that it were running in a virtual machine, then it would presumably not let you start an exam to begin with. Of course, it's possible that it collects all this information and then informs your school, but I haven't heard any reports of this. At least, not yet—this _could_ easily change without any notice, but it's relatively unlikely.
